.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+< A brief description of the pull request >
+
+---
+
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | yes/no
+| Deprecations? | yes/no
+| Tests pass?   | yes/no
+
+---
+
+## Before
+
+< The current status >
+
+## After
+
+< What will be changed >
+
+## Screenshots
+
+< If required. [LICEcap](http://www.cockos.com/licecap/) for Mac or SilentCast for Linux allow you to great GIFs from screen capture >
+
+## Comments
```

Comment:
Alright, I suppose for bigger tickets there could be a lot of content which would fit in here, although for small bug tickets it could be deleted.